src/mixins/Statistics.js

```diff
@@ -0,0 +1,28 @@
+const uuid = `uid_${Math.random().toString(12).slice(2)}`
+export default {
+  methods: {
+    sendStatistics: (benefits, id, event = "display") => {
```

Comment:
```suggestion
    sendStatistics: (benefits, id, event = "displayed") => {
```


```suggestion
    sendStatistics: (benefits, id, event = "show") => {
```


Quelle est la nomenclature actuelle ?

Reply:
Aujourd'hui on a `display` et `clicked` dans le schéma de mongo : https://github.com/betagouv/aides-jeunes-stats-recorder/blob/master/schemas/benefitsRecords.mongoose.js

On part plutôt sur `displayed` du coup ?

Reply:
PR associé sur stats-aides-jeunes : https://github.com/betagouv/aides-jeunes-stats-recorder/pull/3